### ISSUE TITLE
Debug Information Displayed in Log

### ISSUE THREAD

**Issue by Squigglet** [PokemonGoF/PokemonGo-Bot#3663]
### Expected Behavior

Expected behaviour is that when Debugging is turned off, debug information is not displayed in the log.

### Actual Behavior

Debug information is displayed in the log.

### Your config.json (remove your credentials and any other private info)
```
"websocket_server": false,
"walk": 4.16,
"action_wait_min": 1,
"action_wait_max": 4,
"debug": false,
"test": false,
"health_record": true,
"location_cache": true,
"distance_unit": "km",
"reconnecting_timeout": 15,
"evolve_captured": "NONE",
"catch_randomize_reticle_factor": 1,
"catch_randomize_spin_factor": 1,
```

### Steps to Reproduce

Startup the bot.

### Other Information
OS: Linux
Git Commit: f7838b8b431ff36b12f6c0bd643dbf673db34be5
Python Version: (run 'python -V' and paste it here)

![screen shot 2016-08-12 at 10 22 23 am](https://cloud.githubusercontent.com/assets/20935928/17609290/e6b48138-6076-11e6-8f55-7de064c6a76e.png)

**Comment by k4n30** [comment]
Seeing as you didn't bother to fill out the "Other Information" fields I'm going to assume you are on an older commit which has since been fixed. Please update to the newest version and report back if you still have issues, otherwise please close the issue

**Comment by Squigglet** [comment]
Updated my original post. Most recent commit.

**Comment by DiegoRBaquero** [comment]
Happens to me too. On most recent commit on master

**Comment by k4n30** [comment]
Ahhhh you are both on master (might need to add what branch you're on to the issue template).  The latest master commit was 6 hours ago, this was fixed 4 hours ago in the dev branch #3637. Assuming you want to stay on the master branch this will be fixed at the next merge.

Closing as issue is resolved, just waiting for the merge to master to occur

**Comment by k4n30** [comment]
#3665
